performance/tests/browserAndApi.js

```diff
@@ -0,0 +1,276 @@
+      scenarios.browser = {
+        executor: 'constant-vus',
+        exec: 'browserTest',
+        vus: 3,
+        duration: '5m',
+        // adjust the above value as needed if you need to run browser an API scenarios together 
+        options: {
+          browser: {
+            type: 'chromium',
+            headless: false,
+          },
+        },
+      };
+    }
+
+    if (SCENARIOS.includes('load')) {
+      scenarios.api = {
+        executor: 'ramping-vus',
+        exec: 'runIfLoad',
+        // High load
+        stages: [
+          { duration: '2m', target: 1000 },  // Ramp up to 1k VUs in 2 minutes
+          { duration: '3m', target: 2000 },  // Ramp up to 2k VUs in 3 minutes
+          { duration: '2m', target: 3000 },  // Ramp to 3k VUs in 2 minutes
+          { duration: '5m', target: 3000 },  // Hold at 3k VUs for 5 minutes
+          { duration: '2m', target: 0 },     // Ramp down to 0
+        ],
+         // Lower load
+         /*stages: [
+          { duration: '1m', target: 50 },   // ramp to 50 users
+          { duration: '2m', target: 150 },  // ramp to 150 users
+          { duration: '2m', target: 300 },  // ramp to 300 users
+          { duration: '2m', target: 300 },  // hold at 300
+          { duration: '1m', target: 0 },    // ramp down
+        ],*/
+        gracefulRampDown: '1m',
+      };
+    }
+
+    if (SCENARIOS.includes('soak')) {
+      scenarios.api = {
+        executor: 'constant-vus',
+        exec: 'runIfSoak',
+        vus: 50,
+        duration: '1h',
+      };
+    }
+
+    if (SCENARIOS.includes('smoke')) {
+      scenarios.api = {
+        executor: 'shared-iterations',
+        exec: 'runIfSmoke',
+        vus: 1,
+        iterations: 1,
+        maxDuration: '15s',
+      };
+    }
+    if (SCENARIOS.includes('stress')) {
+      scenarios.api = {
+        executor: 'ramping-vus',
+        exec: 'runIfStress',
+        startVUs: 0,
+        stages: [
+          { duration: '1m', target: 1000 },  // ramp to 1k users
+          { duration: '1m', target: 2000 },  // ramp to 2k users
+          { duration: '1m', target: 3000 },  // ramp to 3k users
+          { duration: '2m', target: 3500 },  // push beyond expected peak
+          { duration: '2m', target: 3500 },  // hold to observe system under stress
+          { duration: '1m', target: 0 },     // ramp down
+        ],
+        gracefulRampDown: '30s',
+      };
+    }
+    return scenarios;
+  })(),
+
+  thresholds: {
+    browser_http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    browser_http_req_duration: ['p(90)<200', 'p(95)<500'], // 90% of requests should be below 200ms / 95% of requests should be below 500ms
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(95)<200'], // 95% of requests should be below 200ms
+  },
+};
+//#endregion
+
+//#region browser test
+
+export async function browserTest() {
+  const page = await browser.newPage();
```

Comment:
can we renamed the `page` and `page2` fields to `stubPage` and `sanPage` or something to make them a bit more quickly understandable?